src/main/java/org/jabref/gui/EntryTypeView.java

```diff
@@ -168,4 +183,116 @@ private void setEntryTypeForReturnAndClose(Optional<BibEntryType> entryType) {
         viewModel.stopFetching();
         this.close();
     }
+
+    //The description is coming from biblatex manual chapter 2
+    //Biblatex documentation is favored over the bibtex,
+    //since bibtex is a subset of biblatex and biblatex is better documented.
+    public static String getDescription(EntryType selectedType) {
+        if (selectedType instanceof StandardEntryType entry) {
```

Comment:
Unfortunately you have to change this feature to the old school format, as the new pattern matching is only available as preview.
```suggestion
        if (selectedType instanceof StandardEntryType ) {
        {
        var entry = (StandardEntryType) selectedType;
```